modules/ROOT/pages/super-administrator-application-overview.adoc

```diff
@@ -0,0 +1,73 @@
+= Bonita Super Administrator Application overview
+:description: This pages describes the Bonita _Super Administrator Application_: its values, how to access it, and how to customize it. +
```

Comment:
The "the" sounds weird to me but maybe it is OK. It is just a suggestion. feel free to ignore
```suggestion
:description: This pages describes Bonita's _Super Administrator Application_: its values, how to access it, and how to customize it. +
```

Reply:
We should either capitalize without "the" or not capitalize with "the". You are right. I am making the change.